client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-font-pairings.tsx

```diff
@@ -0,0 +1,55 @@
+import { Button } from '@automattic/components';
+import { FontPairingVariations } from '@automattic/global-styles';
+import { __experimentalNavigatorBackButton as NavigatorBackButton } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import NavigatorHeader from './navigator-header';
+import type { GlobalStylesObject } from '@automattic/global-styles';
+
+interface Props {
+	siteId: number | string;
+	stylesheet: string;
+	selectedFontPairingVariation: GlobalStylesObject | null;
+	onSelect: ( fontPairingVariation: GlobalStylesObject | null ) => void;
+	onDoneClick: () => void;
+}
+
+const ScreenFontPairings = ( {
+	siteId,
+	stylesheet,
+	selectedFontPairingVariation,
+	onSelect,
+	onDoneClick,
+}: Props ) => {
+	const translate = useTranslate();
+
+	return (
+		<>
+			<NavigatorHeader
+				title={ translate( 'Fonts' ) }
+				description={ translate(
+					'We’ve hand picked a selection of font pairings that you can customise later.'
```

Comment:
CMIIW, but we need to use American spelling: customize.

Reply:
Yes. I'll create a follow-up PR to update all copies to American spelling 🙂